packages/sdk-demo/src/apps/SDKDemo/SDKDemo.tsx

```diff
@@ -136,3 +184,42 @@ const MoniteReactQueryDevtools = () => {
   const { queryClient } = useMoniteContext();
   return <ReactQueryDevtools initialIsOpen={false} client={queryClient} />;
 };
+
+const AuthErrorsBackdrop = ({
+  errors,
+  logout,
+}: {
+  errors: Array<Error>;
+  logout: () => void;
+}) => {
+  if (!errors.length) return null;
+
+  return (
+    <Backdrop open>
+      <Stack
+        spacing={2}
+        maxWidth="sm"
+        margin="auto"
+        height="100vh"
+        justifyContent="center"
+      >
+        {errors.map((error) => (
+          <Alert severity="error" key={error.message}>
+            Error: <code>{error.message}</code>
```

Comment:
add localisation pls

Reply:
Didn't want to inflate the size of the dictionary with util messages. Added.